src/renderer/components/+landing-page/landing-page.tsx

```diff
@@ -1,40 +1,24 @@
 import "./landing-page.scss";
 import React from "react";
-import { observable } from "mobx";
 import { observer } from "mobx-react";
-import { clusterStore } from "../../../common/cluster-store";
-import { workspaceStore } from "../../../common/workspace-store";
+import { Workspace, workspaceStore } from "../../../common/workspace-store";
+import { WorkspaceOverview } from "./workspace-overview";
+import { PageLayout } from "../layout/page-layout";
 
 @observer
 export class LandingPage extends React.Component {
-  @observable showHint = true;
 
+  get workspace(): Workspace {
+    return workspaceStore.currentWorkspace;
+  }
```

Comment:
Because it's one less time I have to write out `workspaceStore.currentWorkspace`? :)
I had this class doing more but refactored it out. May add some back after reviews...